packages/design-system/src/components/Button/Button.tsx

```diff
@@ -26,6 +25,10 @@ type CommonButtonProps<T> = {
    */
   component?: T;
   disabled?: boolean;
+  /**
+   * Button [`type`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type) attribute
```

Comment:
This comment doesn't match the `href` prop beneath, does it?

Reply:
Whoops! Good catch